Coypu.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>Coypu</id>
-    <version>2.11.0</version>
+    <version>2.12.0</version>

```

Comment:
Usually a one or more changes are combined into a package release by whoever is maintaining the Nuget package. As such it's better to leave this alone and have just the code changes in this PR